src/services/github.ts

```diff
@@ -181,38 +183,53 @@ const createFileContent = async (folderName, fileName, base64, commitMsg) => {
   }
 }
 
-const changeHeadsCommitSha = async commitSha => {
-  const endpoint = `${ENDPOINT_BASE}/git/refs/heads/master`
-  const bodyData = {
-    sha: commitSha,
-  }
-  try {
-    const result = await fetch(endpoint, {
-      method: 'POST',
-      headers: {
-        Authorization: `token ${GITHUB_API_TOKEN}`,
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify(bodyData),
-    })
-    const data = await result.json()
+type Base64ExtType = {
+  imageExtension: string
+  base64: string
+}
 
-    return { data: data, error: !result.ok }
-  } catch (err) {
-    console.error(`Error requesting tree sha`, err)
-    return { error: true }
+type LinksMetadadaType = {
+  community?: Array<Record<string, unknown>>
+  documentation?: Array<Record<string, unknown>>
+}
+
+type GardenMetadataType = {
```

Comment:
A nitpick, the GardenMetada also has two  other optional values that are: [wiki](https://github.com/1Hive/dao-list/blob/master/xdai.json#L10), and [wrappableToken](https://github.com/1Hive/dao-list/blob/master/xdai.json#L55-L57).

Reply:
Nice catch! On the original code i not see them, i miss? Where i get them? I'll print some logs to check and add them

Reply:
I added

Reply:
This is related to something we were discussing with @tmoutinho . Those fields were included on the `Add profile` step of the garden creation wizard. We were including them manually after the fact. At least for the wrappable token, we should include them in the onboarding.